app/src/main/java/com/yeonae/chamelezone/CheckDialogFragment.kt

```diff
@@ -92,7 +92,7 @@ class CheckDialogFragment : DialogFragment() {
             setOnClickListener {
                 if (this.isChecked) {
                     selectedKeyword.add(this.text.toString())
-                } else if (this.isChecked) {
+                } else if (!this.isChecked) {
```

Comment:
```suggestion
                } else {
```
isChecked==true의 반대는 else뿐일것 같습니다.

Reply:
수정했습니당